addon/models/module.js

```diff
@@ -10,10 +10,10 @@ export default class Module extends Model {
   @attr
   functions;
 
-  @hasMany('class', { async: false, inverse: null })
+  @hasMany('class', { async: false, inverse: null, polymorphic: true })
   classes;
 
-  @hasMany('class', { async: false, inverse: null })
+  @hasMany('class', { async: false, inverse: null, polymorphic: true })
```

Comment:
If someone knowledgeable about this library comes across this, is it normal for 'components' to be a 'class' model rather than a 'component' model ?